src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,29 +4,39 @@
 // </copyright>
 //---------------------------------------------------------------------
 
-using System.Diagnostics.CodeAnalysis;
+using System;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
-    public enum ODataLibraryCompatibility
+    [Flags]
+    public enum ODataLibraryCompatibility : long
     {
+        /// <summary>
+        /// No specific version or feature set.
+        /// </summary>
+        None = 0, // No flags set
+
+        /// <summary>
+        /// When enabled, the CSDL Scale and SRID "variable" value will be written "Variable" instead of "variable" for compatibility with older versions of the library.
+        /// </summary>
+        UseLegacyVariableCasing = 1 << 0,

```

Comment:
@habbes Instead of working out the equivalent int or long value, it's considered easier if the enum flags values are specified as:
```
1 << 0,
1 << 1,
1 << 2,
1 << 3,
1 << 4,
1 << 5,
// ...
1 << 60,
1 << 61,
1 << 62
```

`1 << 0` is just to make it uniform with the other enum values.

Reply:
I get the need for `1 << x` where `x > 0`. Just thought `1 << 0` is superfluous. Like if we did `1 + 0` do be consistent with places where we had `1 + 1`. But not a hill I'm willing to die on :)